src/containers/FileViewers.tsx

```diff
@@ -0,0 +1,389 @@
+import React, { useState, useEffect } from 'react';
+
+// MUI - Components
+import {
+  Paper,
+  Grid,
+  Typography,
+  IconButton,
+  CircularProgress,
+  ButtonBase,
+  InputLabel,
+  MenuItem,
+  FormControl,
+  Select,
+  ImageListItemBar,
+} from '@material-ui/core';
+import { blueGrey, grey } from '@material-ui/core/colors';
+
+// MUI - Icons
+import ChevronLeft from '@material-ui/icons/ChevronLeft';
+import WarningIcon from '@material-ui/icons/Warning';
+import InfoIcon from '@material-ui/icons/Info';
+
+// Other libraries
+import { useParams, Link } from 'react-router-dom';
+import { API } from 'aws-amplify';
+
+// Import dialog details
+import LimsRowDetailsDialog from '../components/LimsRowDetailsDialog';
+
+type pipelineOptionType = {
+  name: string;
+  regexKey: string;
+};
+
+const GPL_PIPELINE_OPTIONS: pipelineOptionType[] = [
+  { name: 'GPL - all', regexKey: 'gridss_purple_linx/(.*)(.png$)' },
+  { name: 'GPL - purple', regexKey: 'gridss_purple_linx/purple/(.*)(.png$)' },
+  { name: 'GPL - linx', regexKey: 'gridss_purple_linx/linx/(.*)(.png$)' },
+];
+
+const UMCCRISED_PIPELINE_OPTIONS: pipelineOptionType[] = [
+  { name: 'UMCCRISED - all', regexKey: 'umccrised/(.*)(.png$)' },
```

Comment:
Can you change display to just `umccrise` without `d`, here pls.

Reply:
Resolved in commit `bd8dc4a`